src/components/figures/figure_container/figure_container.ts

```diff
@@ -284,13 +285,14 @@ export class FiguresContainer extends Component<Props, SpreadsheetChildEnv> {
         this.dnd.y = dndInitialY - deltaY;
       }
 
-      if (this.dnd.x < 0) {
-        this.dnd.width += this.dnd.x;
-        this.dnd.x = 0;
+      // Restrict resizing if x or y reaches header boundaries, and adjust width/height accordingly
```

Comment:
We do not restrict anything, right ?

Reply:
@dhrp-odoo could you look into this comment?